code/game/machinery/teleporter.dm

```diff
@@ -71,8 +71,8 @@
 			if(!calibrated && prob(30 - ((accurate) * 10))) //oh dear a problem
 				log_game("[M] ([key_name(M)]) was turned into a fly person")
```

Comment:
You may want to move this below the dna and rad immunity check, it's still not granted the "fly-ification" will happen at this point yet.